docs/milestone-1-validation.md

```diff
@@ -0,0 +1,97 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault & Cloud Server on a VM and expose the URL and Port
+
+1. Update environment file `application.properties` with Vault target URL (before starting Scality OSIS application)
+    * Vault URL should be set at property `osis.scality.vault.endpoint` (Refer Sample `application.properties` [here](../src/main/resources/application.properties))
+    * Vault Super admin credentials should be updated at properties `osis.scality.vault.access-key` and `osis.scality.vault.secret-key`
```

Comment:
The application.properties might need a comment that we need to use superadmin creds there and not just any AK/SK. But is still fine.

Reply:
Updated the same in c32406aabea800b30939886856300f757611a70d